crates/cxx-qt-lib/src/core/quuid.cpp

```diff
@@ -0,0 +1,71 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
```

Comment:
I think it can just be you as a contributor to this code, right? Unless there's some backstory to this MR I'm not getting